### ISSUE TITLE
Toaster

### ISSUE THREAD

**Issue by liyokuna** [Orange-OpenSource/Orange-Boosted-Angular#89]
To be done when adding Toaster
```
- Checking the contrast
- Checking the accessibility of the toaster
- Checking the compliance design with our brand.
```

**Comment by Aniort** [comment]
accessibility issue : 
WCAG 2.2.1 Timing Adjustable: For each time limit that is set by the content, at least one of the following is true: 
   

-  Turn off: The user is allowed to turn off the time limit before encountering it; or
-     Adjust: The user is allowed to adjust the time limit before encountering it over a wide range that is at least ten times the length of the default setting; or
-     Extend: The user is warned before time expires and given at least 20 seconds to extend the time limit with a simple action (for example, "press the space bar"), and the user is allowed to extend the time limit at least ten times; or
-     Real-time Exception: The time limit is a required part of a real-time event (for example, an auction), and no alternative to the time limit is possible; or
-     Essential Exception: The time limit is essential and extending it would invalidate the activity; or
-     20 Hour Exception: The time limit is longer than 20 hours.

It's not the case in the 2 toasters, so you should implement one of these solution : adjust, extend, turn-ff (see before)

You cannot prevent the toast to disappear when you're on hover with the mouse, and it must work with a keyboard (it's a way to implement turn-off)
We have to think of a solution together !

**Comment by liyokuna** [comment]
#96
